apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -134,7 +134,7 @@ export default function ListBox({
 
   useEffect(() => {
     fetchData();
-  }, [layout]);
+  }, [layout, checkboxes]);
```

Comment:
Same here, it is needed to enforce the new selection state on the fields, since checkbox fields should not have a green/gray background color, we need to trigger a refresh when switching between these two selection modes.

Reply:
Had we triggered checkboxes: true/false from outside (using the Stardust API) we would also have run this effect. But since we are changing selection mode internally (from within the Listbox) we need to trigger this to happen by adding this flag to the effect watch array.